manual/en/webgpurenderer.html

```diff
@@ -0,0 +1,179 @@
+
+             <p>
+             Apart from the fact that `WebGPURenderer` enables access to WebGPU, it offers an exciting feature set:
+             <p>
+
+            <ul>
+              <li>
+                `WebGPURenderer` comes with a new node-based material system which allows to develop custom materials with greater flexibility and 
+                more robustness.
+              </li>
+              <li>
+                The renderer supports TSL, the three.js shading language. With TSL, developers can write shader code with JavaScript in a 
+                platform-independent manner. Shader code written in TSL can be transpiled to WGSL or GLSL depending on the available backend.
+              </li>
+              <li>
+                `WebGPURenderer` comes with a new post-processing stack with built-in Multiple Render Targets (MRT) support and automatic pass combination thanks to the 
+                node material.
+              </li>
+            </ul>
+
+            Let's find out how to integrate `WebGPURenderer` in three.js applications.
+
+            <h2>Usage</h2>
+
+             <p>
+             `WebGPURenderer` has different build files so the way you import three.js changes:
+            </p>
+            
+<pre class="prettyprint showlinemods notranslate lang-js" translate="no">
+-  import * as THREE from 'three';
++  import * as THREE from 'three/webgpu';
+</pre>
+
+            <p>
+              If you are using an import map, it's recommended to change it to the following (the paths differ depending on your setup):
+            </p>
+<pre class="prettyprint showlinemods notranslate lang-js" translate="no">
+  &lt;script type="importmap"&gt;
+    {
+      "imports": {
+        "three": "../build/three.webgpu.js",
+        "three/webgpu": "../build/three.webgpu.js",
+        "three/tsl": "../build/three.tsl.js",
+        "three/addons/": "./jsm/"
+      }
+    }
+  &lt;/script&gt;
+</pre>
+
+             <p>
+              You can create an instance of the renderer just like with `WebGLRenderer`:
+            </p>
+
+<pre class="prettyprint showlinemods notranslate lang-js" translate="no">
+const renderer = new THREE.WebGPURenderer( { antialias: true } );
+renderer.setPixelRatio( window.devicePixelRatio );
+renderer.setSize( window.innerWidth, window.innerHeight );
+renderer.setAnimationLoop( render );
+document.body.appendChild( renderer.domElement );
+</pre>
+            <p>
+              It's important to understand that WebGPU is initialized in an asynchronous fashion. Hence, it is recommended to use 
+              `setAnimationLoop()` to define the animation loop of your app since this approach will automatically ensure the renderer
+              is initialized when rendering the first frame. If you prefer to manage your animation loop via `window.requestAnimationFrame()` or
+              if you have to use to the renderer in your init routine, you need an additional line in the above code section.
```

Comment:
@Mugen87 "you have to use to the renderer", the second "to" is superfluous here no?